wallet/connect/index.md

```diff
@@ -0,0 +1,57 @@
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------:|:------------------------------:|:---------------:|
+| Connect from a web dapp to the MetaMask extension                                                                                                                                                                                                                                                  |      ✓       |               ✓                |        ✓        |
+| Connect from a web dapp, desktop, mobile, and gaming dapps to MetaMask Mobile                                                                                                                                                                                                                      |      ✓       |               ✓                |                 |
+| Connect to MetaMask using the [EIP-6963 provider](../concepts/wallet-interoperability.md)                                                                                                                                                                                                          |      ✓       |               ✓                |        ✓        |
+| Compatible with other [EIP-6963](../concepts/wallet-interoperability.md) wallets                                                                                                                                                                                                                   |              |               ✓                |        ✓        |
+| Custom RPC methods such as [`connectAndSign`](../how-to/sign-data/connect-and-sign.md), [custom modals](../how-to/display/custom-modals.md), [read-only requests](../how-to/make-read-only-requests.md) using Infura API, and [RPC request batching](../how-to/batch-json-rpc-requests.md) |      ✓       |               ✓                |                 |
+
+## MetaMask SDK
+
+MetaMask SDK enables a reliable, secure, and seamless connection between your dapp and the MetaMask browser extension or MetaMask Mobile. You can call any Wallet API method from a dapp with the SDK installed, and you can integrate the SDK into existing projects without modifying existing Wallet API calls.
+
+Key features include:
+
+- **Multi-platform support** – Connect from web, desktop, mobile, and gaming platforms.
+- **Seamless mobile integration** – Use QR codes or deeplinks to establish persistent connections with MetaMask Mobile.
+- **EIP-6963 detection** – Automatically detect MetaMask, with or without third-party libraries.
```

Comment:
Instead of calling functionality be it's EIP number, should we rather call it by what it does/provides (or EIP name)?
There are so many EIPs going around, and likely more that will be added in the future, so could become confusing. Perhaps something we should consider in the future (don't need to address now).

Reply:
Updated the descriptions to say "Automatically detect MetaMask without conflicting with other installed wallets."

Reply:
I can confirm with the SDK team, but it seems like many existing developers do look for the EIP number for certain major features (6963 being one of them).